cartographer/mapping_2d/pose_graph.cc

```diff
@@ -320,6 +320,11 @@ void PoseGraph::HandleWorkQueue() {
         for (auto& trimmer : trimmers_) {
           trimmer->Trim(&trimming_handle);
         }
+        trimmers_.erase(std::remove_if(
+            trimmers_.begin(), trimmers_.end(),
+            [](std::unique_ptr<mapping::PoseGraphTrimmer>& trimmer) {
+              return trimmer->IsFinished();
+            }));
```

Comment:
Shouldn't this be

~~~
trimmers_.erase(std::remove_if(
          trimmers_.begin(), trimmers_.end(),
          [](std::unique_ptr<mapping::PoseGraphTrimmer>& trimmer) {
            return trimmer->IsFinished();
          }),
    trimmers_.end());
~~~